lib/foreman_discovery/engine.rb

```diff
@@ -65,9 +65,9 @@ class Engine < ::Rails::Engine
             :discovered_hosts          => [:update_multiple_location,
                                            :select_multiple_organization,
                                            :update_multiple_organization,
-                                           :select_multiple_location, :refresh_facts, :reboot, :reboot_all],
+                                           :select_multiple_location, :refresh_facts, :reboot, :multiple_reboot],
             :hosts                     => [:process_hostgroup],
-            :"api/v2/discovered_hosts" => [:refresh_facts, :reboot, :reboot_all]
+            :"api/v2/discovered_hosts" => [:refresh_facts, :reboot, :multiple_reboot]
```

Comment:
In API context there is no multiple support, our API can only reboot once host and reboot all. Please keep V2 API unchanged in this regard.

We are actually missing reboot_all CLI command from our CLI, so only API for reboot_all and auto_prov_all can be used, not CLI. If you want go ahead and create these options (`--all`) in https://github.com/theforeman/hammer-cli-foreman-discovery